app/src/main/java/br/org/cesar/madmovies/movies/domain/model/Movie.kt

```diff
@@ -1,13 +1,9 @@
 package br.org.cesar.madmovies.movies.domain.model
 
-import android.os.Parcelable
-import br.org.cesar.madmovies.movies.data.model.RemoteMovie
-import br.org.cesar.madmovies.movies.data.model.RemoteMovieList
-import kotlinx.android.parcel.Parcelize
-
-@Parcelize
-data class Movie(val id: Int, val title: String, val year: String) : Parcelable {
-    constructor(movie: RemoteMovie) : this(
-        movie.id, movie.title, movie.release_date ?: ""
-    )
+data class Movie(val id: Int,
```

Comment:
The Kotlin code convention advices to break line after `(` and have closing `)` in new line. Of course, your project can follow whichever convention it prefers as long as it is consistent.

Tip: you can apply Kotlin code conventions on Android Studio / IntelliJ editor settings and reformat automatically with command + shift + L

Reply:
I couldn't apply this shortcut. And couldn't find it on Settings either. :(

Reply:
fixed

Reply:
It's actually command option L :)